app/logic/events.py

```diff
@@ -446,3 +446,10 @@ def getTomorrowsEvents():
     tomorrowDate = date.today() + timedelta(days=1)
     events = list(Event.select().where(Event.startDate==tomorrowDate))
     return events
+def sortEvents(EventList):
```

Comment:
Lets try and make this name more descriptive. What events are being sorted or why are the events being sorted?

Reply:
This issue still needs resolved @qasema